### PR TITLE
Expose by day totals

### PR DESCRIPTION
**Pull request by adamlofting**
@jbuck R?

I'm not setup with a local dev environment and creds for this so I've not run it, but thought I'd take a pass at the code.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -33,6 +33,20 @@ var bycountry_query = `SELECT country_code, sum(total)::numeric AS total, sum(do
                        GROUP BY country_code
                        ) AS bycountry GROUP BY bycountry.country_code ORDER BY bycountry.country_code`;
 
+
+var byday_query = `SELECT day, SUM(total) AS total FROM
+                  (SELECT SUM(settle_amount)::numeric as total, DATE(timestamp) as day FROM paypal
```
Comment (jbuck) [inline server.js]: I'd change the `DATE(timestamp)` function to `date_trunc('day', timestamp)` in the various parts of this code

**Comment by jbuck** [comment]
@adamlofting I cheat and use the production database for testing all my patches because I'm a lazy fuck. Anyways this looks fine to me, I'd just make that date_trunc change

**Comment by adamlofting** [comment]
Done. I'll let you merge, in case there are any deploy mechanisms hooked up and I'm not able to fix right now.

**Comment by jbuck** [comment]
It is set to auto deploy on merges to master, and I added you to the app yesterday so you can manage it too!

**Pull request merged**